Makefile

```diff
@@ -488,7 +488,7 @@ $(DOCKERFILE_FROM_CHECKER): $(DOCKERFILE_FROM_CHECKER_DIR)/*.go $(DOCKERFILE_FRO
 # this next section checks that the FROM hashes for any image in any dockerfile anywhere here are consistent.
 # For example, one Dockerfile has foo:abc and the next has foo:def, it will flag them.
 # These are the packages that we are ignoring for now
-IGNORE_DOCKERFILE_HASHES_PKGS=alpine
+IGNORE_DOCKERFILE_HASHES_PKGS=alpine installer
```

Comment:
@rene don't we need to add monitor as well? it will be checked against installer if I understand the logic correctly

Reply:
or in case of monitor we just collect a list of all packages that uses package X and just check they all the same?

Reply:
@rucoder , good question, I won't add pkg/monitor because it has `FROM lfedge/eve-alpine` if we do wanna check... @christoph-zededa , how it should be done in this case?